core/runtime/src/main/java/io/quarkus/runtime/ResourceHelper.java

```diff
@@ -13,6 +13,7 @@
 public class ResourceHelper {
 
     public static void registerResources(String resource) {
+        resource = resource.replace('\\', '/')); // correct Windows paths
```

Comment:
this probably fixes it but why is the resource using non portable paths to begin with? should be platform neutral before it gets here.

Reply:
You guys are the experts 😄

Reply:
@zakkak will you have a look at this issue?

Reply:
Sure, but the sooner I will probably be able to have a look at it will be late next week.

In the meantime, #31185 is now merged, @melloware could you please confirm the issue is still present in Quarkus `main`?

Reply:
I am running on 2.16.3 not on SNAPSHOT so I have not confirmed it in main yet.  Will try to get to it.